modules/nf-core/cnvkit/antitarget/main.nf

```diff
@@ -2,7 +2,7 @@ process CNVKIT_ANTITARGET {
     tag "$meta.id"
     label 'process_low'
 
-    conda "bioconda::cnvkit=0.9.9"
+    conda "bioconda::cnvkit=0.9.9 bioconda::samtools=1.16.1"
```

Comment:
something 🐟 . Only samtools in conda but not in the container declaration

Reply:
ok, so from @adamrtalbot :
> It's pysam's fault, I think. It isn't explicit about which htslib it needs, which causes a bit of a cascade. I think you can solve it using conda-forge::openssl==version but not certain and it doesn't work on my machine right now. This seems like the best compromise I can come up with quickly.

cf https://github.com/nf-core/modules/pull/2811

Reply:
:shake-angry-fist: well then... :D